packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -5,6 +5,7 @@ import {
   days,
 } from '../../../../react-components/src/helpers/options-for-select.js';
 
+import { isFullDate } from '../../../../react-components/src/helpers/validations.js';
```

Comment:
We probably shouldn't be importing things like this from the react components. That could cause problems when we go to delete the React component, if whoever is doing that work doesn't know these imports are here. Could we copy these imports to a `date-utils.ts` file in the web components `utils` directory? Sorry I overlooked the months and days imports in my initial review.

Reply:
@k80bowman updates have been made and the `date-utils.ts` file has been created